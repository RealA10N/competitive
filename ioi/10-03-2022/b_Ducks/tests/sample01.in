3
-7 7 3
0 2 3
-4 2 1
3 3 1
