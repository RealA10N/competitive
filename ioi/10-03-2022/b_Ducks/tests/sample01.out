5
12
15
15
